.github/workflows/docs-workflow.yml

```diff
@@ -118,3 +118,17 @@ jobs:
             echo "Found broken symlinks: $BROKEN_SYMLINKS"
             exit 1
           fi
+
+
+  check-markdown-links:
+    needs: path_filter
+    if: ${{ needs.path_filter.outputs.should_not_skip == 'true' }}
+    name: Check for broken URLs in markdown files
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3.1.0
+      - uses: gaurav-nelson/github-action-markdown-link-check@5c5dfc0ac2e225883c0e5f03a85311ec2830d368 # pin@v1
```

Comment:
Nit: Perhaps it would be better to use `d53a906aa6b22b8979d33bc86170567e619495ec`/`1.0.15` ?

Reply:
Yeah looks like the new version does not produce these errors anymore